data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"The faster we can get the book published, the better. Meet me in the spaceport in a few hours; I'll find a good company."`
+			`	Before Anna can head off, the Heliarch rises. "Again accompany you, I will."`
+
+
+mission "Coalition: Struggling Author 4"
+	name "Kimek Publishing Company"
+	description "Anna found a Kimek publishing company who is willing to take in Anna Fangvin to get her book published. Bring Anna Fangvin to <destination>."
+	source "Ki Patek Ka"
+	destination "Inmost Blue"
+	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
+	passengers 2
+	to offer
+		has "Coalition: Struggling Author 3: done"
+	on offer
+		conversation
+			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being one of the few humans within the Coalition makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`
+			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <planet>," explains Anna, once again enthusiastic.`
+			choice
+				`	"Great!"`
+					goto go
+				`	"Let's go!"`
+					goto go
+				`	"What's the publishing company called?"`
+			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`
+			label go
+			`	Anna heads to your ship, and you plot a course to <destination>. Just before you launch, the Heliarch hops in.`
+				launch
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive.' She hastily opens the hatch as soon as it unlocks.`
+			`	Anna stops, waiting for you to catch up. "In the message they sent me, they included instructions on how to reach their offices," she explains. "Let's go!"`
+			`	You begin following Anna to the publisher's offices. You walk through the spaceport and its endless rows of shops and streets, eventually approaching what looks like a train station. Anna looks at the message again and tells you which platform to go to and which train to take. After a few minutes the train arrives. You and Anna board the train and travel to a neighborhood where a large white building dominates the skyline.`
+			`	Once the train stops, you get off onto the platform and exit the station. Circling around to the publisher's office, you find the entrance bustling with Kimek, along with the occasional Saryd and Arach coming and going. Anna looks for a front desk and directs you to a Kimek who looks appears to be a receptionist.`
```

Comment:
The Saryd Heliarch isn't really acknowledged here. I would think that he'd be more capable of guiding you three through the city and to the publishing company. Adding mention of him being present later in the interview for the book would also be good.